tests/test_wave_05.py

```diff
@@ -120,29 +140,34 @@ def test_delete_goal(client, one_goal):
     assert response.status_code == 404
 
     response_body = response.get_json()
-    assert "message" in response_body
+    assert "details" in response_body
+    assert response_body == {"details": "Goal 1 not found"}
```

Comment:
I think its okay to remove the first `assert` statement since you have the second one that implicitly checks for `"details"` being a key in the dictionary.